roles/nfs-server/tasks/configure_lvm.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: "Set VG name"
+  set_fact:
+    nfs_vg_name: "{{ nfs_vg_name | default(default_nfs_vg_name) }}"
+
+- name: "Set LV name"
+  set_fact:
+    nfs_lv_name: "{{ nfs_lv_name | default(default_nfs_lv_name) }}"
+
+- name: "Set NFS share basename"
+  set_fact:
+    nfs_share_basedir: "{{ nfs_share_basedir | default(default_nfs_share_basedir) }}"
+
+- name: "Setup PV"
+  shell: "pvcreate {{ nfs_storage_device }}" 
+  when: lvm_check.rc != 0
+
+- name: "Setup VG"
+  shell: "vgcreate {{ nfs_vg_name }} {{ nfs_storage_device }}"
+
+- name: "Setup LV"
+  shell: "lvcreate -n {{ nfs_lv_name }} -l 100%VG {{ nfs_vg_name }}"
+
+- name: "Create file system on share"
+  shell: "mkfs.xfs /dev/mapper/{{ nfs_vg_name }}-{{ nfs_lv_name }}"
+
+- name: "Make dir for NFS shares" 
+  file:
+    path: "{{ nfs_share_basedir }}"
```

Comment:
We could, but the system default is good enough here so prefer to just leave it as-is for now.